buildchain/buildchain/constants.py

```diff
@@ -93,4 +102,21 @@ def load_version_information() -> None:
 SHORT_VERSION : str = '{}.{}'.format(VERSION_MAJOR, VERSION_MINOR)
 VERSION : str = '{}.{}{}'.format(SHORT_VERSION, VERSION_PATCH, VERSION_SUFFIX)
 
+
+def git_ref() -> Optional[str]:
```

Comment:
We may want to factor our this with `git_revision` from `buildchain/buildchain/iso.py` (or only keep one, are they seems similar enough to be confusing and looks like two different way to get the same info)

Can be done here in a dedicated commit or in another PR (then a debt ticket should be created).

Reply:
Done